source/getting-started/use_cases/regulatory_compliance.rst

```diff
@@ -97,11 +97,28 @@ Expand the output to see an example alert for rule ``Apache: Attempt to access f
     "timestamp": "2020-09-04T06:08:53.878+0000"
   }
 
-Example of regulatory compliance dashboard for PCI DSS:
-
-.. thumbnail:: ../../images/getting_started/use_case_regulatory_compliance.png
-   :align: center
-   :wrap_image: No
+Example of regulatory compliance dashboards:
```

Comment:
These images need to be edited. I can see the black line from the header you removed.

Also, I can tell images in this class have different sizes. Please, ask the design team to adjust them properly. If this is too complicated for them to change, we can discuss it we should leave them as it is. 


![image](https://user-images.githubusercontent.com/79285586/141322991-dd6bfb16-9627-4f25-9744-a130aa23229f.png)